content/docs/secure/encryption/rekey.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Rekey Configuration"
+linkTitle: "Rekey Configuration"
+weight: 4
+Description: >
+  Rekey Configuration and Usage
+---
+
+## Rekey Controller Installation
+
+The CSM Encryption Rekey CRD Controller is an optional component that, if installed, allows encrypted volumes rekeying in a
+Kubernetes cluster. The Rekey Controller can be installed via the standard Dell Helm Chart repository available 
+at https://github.com/dell/helm-charts.
+
+A secret of the cluster config must be created with the name ``cluster-kube-config`` typically from the .kube/config. Here is an example:  
+
+```shell
+ kubectl create secret generic cluster-kube-config --from-file=/home/root/.kube/config
+```
+
+The Rekey Controller Helm chart defines these values:
+
+
+```yaml
+# Rekey controller image name.
+image: dellemc/csm-encryption-rekey-controller:v0.1.0
+
+# Rekey controller image pull policy.
+# Allowed values:
+#  Always: Always pull the image.
+#  IfNotPresent: Only pull the image if it does not already exist on the node.
+#  Never: Never pull the image.
+imagePullPolicy: IfNotPresent
+
+# logLevel: Log level of the rekey controller.
+# Allowed values: "error", "warning", "info", "debug", "trace".
+logLevel: "info"
+
+# This value is required and must match encryption.pluginName value
+# of the corresponding Dell CSI driver.
+provisioner:
+
+# This value is required and must match encryption.apiPort value
+# of the corresponding Dell CSI driver.
+port:
+```
+
+| Parameter | Description | Required | Default |
+| --------- |-------------|----------|--|
+| image | Rekey controller image name. | No | dellemc/csm-encryption-rekey-controller:v0.1.0 |
+| imagePullPolicy | Rekey controller image pull policy. | No | IfNotPresent |
+| logLevel | Log level of the rekey controller. | No | info |
+| provisioner | This value is required and must match `encryption.pluginName` value of the corresponding Dell CSI driver. | Yes |  |
+| port | This value is required and must match `encryption.apiPort` value of the corresponding Dell CSI driver. | Yes |  |
+
+## Deploy Rekey Controller
+
+Copy the values.yaml to a local file. Once the CSM Encryption Rekey Controller local values.yaml file has been
+adjusted for the current cluster, deploy the controller by installing the Helm chart. As an example:
+
+``` helm install --values local-values.yaml rekey-controller csm-encryption-rekey-controller```

```

Comment:
If this is installing the helm chart from our official Dell helm repository, then you can also include the step to add the repository similar to step 3 here (`helm repo add dell https://dell.github.io/helm-charts`): https://dell.github.io/csm-docs/docs/observability/deployment/helm/#install-the-csm-for-observability-helm-chart

And the helm install will need to prefix the chart name with `dell`:
```helm install --values local-values.yaml rekey-controller dell/csm-encryption-rekey-controller```